pkg/kubectl/describe.go

```diff
@@ -1395,6 +1396,32 @@ func DescribeEvents(el *api.EventList, w io.Writer) {
 	}
 }
 
+// DeploymentDescriber generates information about a deployment.
+type DeploymentDescriber struct {
+	client.Interface
+}
+
+func (dd *DeploymentDescriber) Describe(namespace, name string) (string, error) {
+	d, err := dd.Experimental().Deployments(namespace).Get(name)
```

Comment:
If we really want them, then instead of recomputing them here, I would prefer that deployment controller adds them to the DeploymentStatus and we just display it here.
I think we had discussed doing that in the proposal, but then decided against it.